corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/menu.less

```diff
@@ -15,7 +15,12 @@
   #module-case-list-map {
     height: calc(~"100vh - 65px");
     position: sticky !important; // Has to be important so the leaflet code does not set it to relative
-    top: 50px; // based on the height of the crumbs bar
+    top: 47.125px; // based on the height of the crumbs bar
+    z-index: 1;
```

Comment:
What happens without z-index set to 1? 

The [design guide ](https://www.commcarehq.org/styleguide/atoms/ )suggests avoiding modifying z-index if possible, otherwise defining it as a constant [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less)

Reply:
Ah, the issue is it sits under the case list and is hidden/unclickable without it, so it just needs to be set to something more than 0. My thought is that since it's just set on a single (non-global) element it should be okay to leave it here and add a comment, what do you think?

Reply:
Ya, I think it's fine since it's applied very specifically, not blocking.

Just wondering if there's a way to avoid it if possible. Is it "under" the case list because of `position: sticky` and only when in the small screen view?

Reply:
It is due to the `position: sticky` that it sits under the case list -- when they collide, the case list takes precedence. It does so I believe because the case list rows are nested a few layers further down in the DOM than the map pane is (meaning they stack that many layers higher with no z-index defined). Technically they're always layered like this since the map is sticky on larger screens as well, there's just not a situation where they would collide like in a small screen. I can't really think of a good alternative solution.